ls1intum/Artemis
# `Exam mode`: Fix an issue with quizzes in test exams

Quizzes were not working any more for test exams, this PR should fix it

### Checklist
#### General



### Steps for Testing

Prerequisites:
- 1 Instructor
- 1 Student
- 1 Test Exam and 1 Real Exam with quizzes

1. Create the test and real exam
2. Participate and make sure quizzes are loaded correctly
3. Submit the test exam and make sure the quiz submission is displayed correctly

### Review Progress





#### Code Review
#### Manual Tests
